emmett.js

```diff
@@ -478,7 +478,8 @@
             this._handlersComplex :
             this._handlersAll;
 
-        parent.splice(parent.indexOf(onces[j]), 1);
+        const index = parent.indexOf(onces[j]);
+        index !== -1 && parent.splice(index, 1);
```

Comment:
Hello @iaspin92. Can you just turn this line into a real condition for readability, please?